src/components/common/TmBalance.vue

```diff
@@ -457,6 +479,55 @@ export default {
 }
 </script>
 <style scoped>
+select {
+  background: var(--input-bg);
+  color: var(--txt, #333);
+  border: none;
+}
+
+select option {
+  background: var(--app-bg);
+  color: var(--txt);
+  font-family: var(--sans);
+}
+
+.currency-flag {
+  width: 15px;
+  margin-right: 3px;
+}
+
+.currency-selector {
+  display: flex;
+  align-items: center;
+}
+
+.total-fiat-value {
+  min-width: 30px;
+  margin-top: 0.25rem;
+}
+
+.user-box {
+  font-size: 12px;
+  margin-right: 0.5rem;
+  padding: 0.25rem 0.5rem;
+  background-color: var(--bc);
+  color: var(--link);
+  border-radius: 1.25rem;
+  display: inline-block;
+  cursor: pointer;
+}
+
+.user-box:hover {
+  color: var(--link-hover);
+}
+
+.value-div {
```

Comment:
maybe this class could be more descriptive? what value is it actually for?

Reply:
this is each currency  block (staking denom on the left and the alt-tokens ones on the right).

Should be called `currency-div`, now that I think about it 🤔